src/components/list/list.ts

```diff
@@ -20,32 +24,30 @@ import {ItemSlidingGesture} from '../item/item-sliding-gesture';
  *
  */
 @Directive({
-  selector: 'ion-list'
+  selector: 'ion-list',
+  host: {
+    '[class.reorder-enabled]': '_enableReorder', 
+  }
 })
 export class List extends Ion {
+  private _enableReorder: boolean = false;
   private _enableSliding: boolean = false;
+  private slidingGesture: ItemSlidingGesture;
```

Comment:
It's half and half `_` and without `_`. Let's just stick with using `_` for private vars since this component could be injected by the user.